Appraisals

```diff
@@ -1,13 +1,13 @@
 appraise "rails-6.0" do
-  gem 'rails', '~> 6.0'
+  gem 'rails', '6.0.6.1'
```

Comment:
Any issue with leaving as `'~> 6.0'`?  
Your README.md requirements show Rails >=6.0.

Reply:
yes. for some reason, leaving the `gem 'rails'` as `'~> 6.0'` will make the `gemfile.lock` to generate Rails 6.1.7.7 instead of 6.0.6.1. We wanted to test 6.0 here, so that's why I put more precise version in here. Also, this version of 6.0.6.1 is guaranteed to be the latest one